packages/preview/not-tudabeamer-2023/0.1.0/template/main.typ

```diff
@@ -0,0 +1,56 @@
+#import "@preview/touying:0.4.2": utils
+#import "@preview/not-tudabeamer-2023:0.1.0": register
+
+#let tuda_logo = read("tuda_logo.svg")
```

Comment:
Would you like to include this logo file in the template?

Reply:
Unfortunately according to our university website this is not easily possible.
> The TU Darmstadt logo is registered as a word and image mark with the German Patent and Trademark Office and enjoys special protection.

Reply:
OK.